tortoise/verifying.go

```diff
@@ -43,16 +43,12 @@ func (v *verifying) processLayer(logger log.Log, lid types.LayerID, ballots []to
 	)
 }
 
-func (v *verifying) verifyLayers(logger log.Log) types.LayerID {
-	localThreshold := weightFromUint64(0)
-	// TODO(dshulyak) expected weight for local threshold should be based on last layer.
-	localThreshold = localThreshold.add(v.epochWeight[v.processed.GetEpoch()])
-	localThreshold = localThreshold.fraction(v.LocalThreshold)
+func (v *verifying) verify(logger log.Log) types.LayerID {
+	localThreshold := computeLocalThreshold(v.Config, v.epochWeight, v.processed)
 
 	for lid := v.verified.Add(1); lid.Before(v.processed); lid = lid.Add(1) {
 		// TODO(dshulyak) expected weight must be based on the last layer.
-		threshold := computeExpectedWeight(v.epochWeight, lid, v.processed)
-		threshold = threshold.fraction(v.GlobalThreshold)
+		threshold := computeGlobalThreshold(v.Config, v.epochWeight, lid, v.processed)
```

Comment:
since you name it `computeGlobalThreshold`, i think it's better to add local threshold inside that method per definition.

Reply:
i thought that for clarity there will be global threshold, local threshold and the threshold (which is a sum of both). you think it will be more accurate to have only first and second, and the first is a sum of both?

Reply:
i thought that was the definition of `TG` in https://community.spacemesh.io/t/uncounted-weight-and-the-global-threshold/228 in all cases. TG now always include TL all the time.

Reply:
you are right, it will be accurate to have only global and local. i will update it in next changeset, as i am optimizing it to work for a rerun with full weight